store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -65,14 +67,17 @@ class FlowStoreTest {
                 )
             )
         pipeline.stream(StoreRequest.cached(3, refresh = false))
+            .take(1) // TODO remove when Issue #59 is fixed.
```

Comment:
this passes w/ the fix but i'm first merging this change since it is independent of the fix.